src/common/components/home/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, VFC } from 'react';
+import SocialLinks from '../elements/SocialLinks';
+
+const Footer: VFC = () => {
+    return (
+        <footer className="bg-gray-800 p-10">
+            <div className="flex flex-row flex-wrap justify-between">
```

Comment:
is this combo of flex and grid (see nested elements) good practice?

Reply:
yeah its fine to have a parent flex element with a grid child